meson.build

```diff
@@ -9,9 +9,27 @@ project('gnome-hexgl', 'c',
 
 gthree = dependency('gthree-1.0')
 
+prefix = get_option('prefix')
+datadir = join_paths(prefix, get_option('datadir'))
+
 cc = meson.get_compiler('c')
 libm = cc.find_library('m', required : false)
 
-sources = ['src/hexgl.c', 'src/utils.c', 'src/analysismap.c', 'src/camerachase.c', 'src/shipcontrols.c', 'src/shipeffects.c', 'src/shaders.c', 'src/particles.c', 'src/hud.c']
+sources = [
+  'src/hexgl.c',
+  'src/utils.c',
+  'src/analysismap.c',
+  'src/camerachase.c',
+  'src/shipcontrols.c',
+  'src/shipeffects.c',
+  'src/shaders.c',
+  'src/particles.c',
+  'src/hud.c'
```

Comment:
I think this conflicts with other changes i made in master

Reply:
reverted